ansible_collections/arista/avd/roles/eos_designs/templates/documentation/fabric-topology.j2

```diff
@@ -1,4 +1,4 @@
-Node Type,Node,Node Interface,Peer Type,Peer Node,Peer Interface
+Node Type,Node,Node Interface,Peer Type,Peer Node,Peer Interface,Enabled/Shutdown
```

Comment:
```suggestion
Node Type,Node,Node Interface,Peer Type,Peer Node,Peer Interface,Enabled
```

Reply:
or if you prefer Shutdown, fine with me as well, just remove the `not` from line 14 suggestion

Reply:
Fixed in my latest push. Making it clear with "Node Interface Enabled" as to not confuse with the "Peer Interface" state